client/src/components/userReviews/UserReviews.jsx

```diff
@@ -1,10 +1,17 @@
 import React from 'react';
-import UserReviewsControl from './UserReviewsControl';
+import ReviewRender from './ReviewRender';
 import classes from './UserReviews.module.css';
 
-function UserReviews({reviews}) {
+function UserReviews({ reviews }) {
```

Comment:
for consistency pick one style of either having the `export default function` all in one line or end your file with `export default`